test/testsimplifytypedef.cpp

```diff
@@ -2544,6 +2545,73 @@ class TestSimplifyTypedef : public TestFixture {
         ASSERT_EQUALS(exp, tok(code, false));
     }
 
+    void simplifyTypedef129() { 
+        {
+           const char code[] = "class c {\n"
+                              "  typedef char foo[4];\n"
+                              "  foo &f ;\n"
+                              "};";
+
+            const char exp [] = "class c { char ( & f ) [ 4 ] ; } ;";
+            ASSERT_EQUALS(exp, tok(code, false));
+        }
+
+        {
+           const char code[] = "class c {\n"
+                              "  typedef char foo[4];\n"
```

Comment:
I don't think this is C++ code. At least g++ 8.3.0 does not like it when I test. Which compiler do you use?

    123.cpp:3:9: error: expected ‘;’ at end of member declaration
       foo & f const;
             ^

Reply:
Indeed. My brain switched off when I tried to add some more test cases. /o\
I'll clean this up.